chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/TriggerDeveloperBanJob.java

```diff
@@ -139,8 +141,18 @@ private String createHtmlEmailBody(JobExecutionContext jobContext) {
             reasonForListingChange = "<strong>ONC-ACB provided reason for listing change:</strong> \""
                     + reasonForListingChange + "\"";
         }
-        int openNcs = updatedListing.getCountOpenNonconformities();
-        int closedNcs = updatedListing.getCountClosedNonconformities();
+        int openSurveillanceNcs = updatedListing.getCountOpenNonconformities();
+        int closedSurveillanceNcs = updatedListing.getCountClosedNonconformities();
+        int openDirectReviewNcs = 0;
+        int closedDirectReviewNcs = 0;
+        try {
+            ListingSearchResult listingSearchResult = searchService.findListing(updatedListing.getId());
+            openDirectReviewNcs = listingSearchResult.getOpenDirectReviewNonConformityCount();
+            closedDirectReviewNcs = listingSearchResult.getClosedDirectReviewNonConformityCount();
+        } catch (InvalidArgumentsException ex) {
+            LOGGER.error("No listing was found with the ID " + updatedListing.getId() + " fro the ListingSearchService. The direct review non-conformity counts are unknown.");
```

Comment:
```suggestion
            LOGGER.error("No listing was found with the ID " + updatedListing.getId() + " from the ListingSearchService. The direct review non-conformity counts are unknown.");
```

Reply:
If we log this error, because DRs are down when the email is generated, I think we should put something in the email that also says something about 0/0 not being "real" data